app/routes/codelists-management.js

```diff
@@ -0,0 +1,10 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class CodelistsManagementRoute extends Route {
+  @service() session;
```

Comment:
nitpicky but the `()` aren't needed here and save an extra function call :smile: